core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -620,7 +620,8 @@ public long getUsedBytes() {
         throw new InvalidArgumentException("Unrecognized worker range: " + workerRange);
     }
 
-    List<WorkerInfo> workerInfoList = new ArrayList<>();
+    List<WorkerInfo> workerInfoList = new ArrayList<>(
+        selectedLiveWorkers.size() + selectedLostWorkers.size());
```

Comment:
Isn't this a duplicate of #14728 ?

Reply:
Oops my branch messed up and I didn't notice. Will clean this up.

Reply:
Removed that commit from my branch.